po/de.po

```diff
@@ -1188,13 +1230,30 @@ msgid ""
 "to select among several online or local services from/to where you can "
 "import or export your tasks."
 msgstr ""
+"❗ Achtung: Seit GTG 0.4 wurden die Erweiterungen zu den "
+"Synchronisierungsdienste deaktiviert bis Jemand (Sie?) repariert und pflegt, "
+"weil diese noch nicht mit der neuen Code funktionieren, oder eine veraltete "
+"API vom entsprechenden Dienst nutzen, die aktualisiert werden soll.\n"
+"\n"
+"Synchronisierungsdienste erlauben GTG, Aufgaben, Notizen oder Fehler von "
+"bestimmten Webseiten oder Programme wie Remember the Milk, Tomboy, "
+"LaunchPad, etc. zu synchronisieren (oder zu im- oder exportieren).\n"
```

Comment:
" etc." instead of ", etc.". The comma makes no sense, see https://de.wikipedia.org/wiki/Et_cetera#Verwendung